examples/examples/attrs-literal-field-names.rs

```diff
@@ -0,0 +1,22 @@
+#![deny(rust_2018_idioms)]
+
+use tracing::{debug, info, span, Level};
+use tracing_attributes::instrument;
+
+#[instrument]
+#[inline]
+fn suggest_band() -> String {
+    debug!("Suggesting a band.");
+    String::from("Wild Pink")
+}
+
+fn main() {
+    let subscriber = tracing_subscriber::fmt()
+        .with_env_filter("attrs_basic=trace")
```

Comment:
This filter won't enable anything, since the name of this module is `attrs-literal-field-names`:
```suggestion
        .with_env_filter("attrs_literal_field_names=trace")
```

Reply:
done